great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -419,3 +421,172 @@ def _build_numeric_rule() -> Rule:
+            mostly=f"{VARIABLES_KEY}mostly",
+            strict_min=f"{VARIABLES_KEY}strict_min",
+            strict_max=f"{VARIABLES_KEY}strict_max",
+            meta={
+                "profiler_details": {
+                    "column_min_values_range_estimator": f"{column_min_values_range_parameter_builder_for_validations.fully_qualified_parameter_name}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY}",
+                    "column_max_values_range_estimator": f"{column_max_values_range_parameter_builder_for_validations.fully_qualified_parameter_name}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY}",
+                },
+            },
+        )
+
+        # Step-5: Instantiate and return "Rule" object, comprised of "variables", "domain_builder", "parameter_builders", and "expectation_configuration_builders" components.
+
+        variables: dict = {
+            "mostly": 1.0,
+            "strict_min": False,
+            "strict_max": False,
+            "bins": 10,
+            "false_positive_rate": 0.05,
+            "quantile_statistic_interpolation_method": "auto",
+            "estimator": "bootstrap",
+            "n_resamples": 9999,
+            "random_seed": None,
+            "include_estimator_samples_histogram_in_details": False,
+            "truncate_values": {
+                "lower_bound": None,
+                "upper_bound": None,
+            },
+            "round_decimals": 1,
+        }
+        parameter_builders: List[ParameterBuilder] = [
+            column_histogram_metric_multi_batch_parameter_builder_for_metrics,
+            column_column_quantile_values_metric_multi_batch_parameter_builder_for_metrics,
+            column_min_metric_multi_batch_parameter_builder_for_metrics,
+            column_max_metric_multi_batch_parameter_builder_for_metrics,
+            column_median_metric_multi_batch_parameter_builder_for_metrics,
+            column_mean_metric_multi_batch_parameter_builder_for_metrics,
+            column_standard_deviation_metric_multi_batch_parameter_builder_for_metrics,
+        ]
+        expectation_configuration_builders: List[ExpectationConfigurationBuilder] = [
+            expect_column_min_to_be_between_expectation_configuration_builder,
+            expect_column_max_to_be_between_expectation_configuration_builder,
+            expect_column_values_to_be_between_expectation_configuration_builder,
+        ]
+        rule: Rule = Rule(
+            name="datetime_columns_rule",
+            variables=variables,
+            domain_builder=datetime_column_type_domain_builder,
+            parameter_builders=parameter_builders,
+            expectation_configuration_builders=expectation_configuration_builders,
+        )
```

Comment:
Maybe a helper to create each one of these fields and then have this parent method do the aggregation and invocation of the `Rule` object?

Reply:
@cdkini I see the idea, but I feel that this "indirection" will, while create a shorter and simpler decomposition, will take a way from the easily recognizing the theme that of all of these components are belonging to.  The way it is makes it easier in terms of copy/paste/modify for other themes (or business rules -- implemented as our `Rule` instances), and most (if not all) others will actually be shorter.  Thanks!